packages/elements/src/slider/elements/slider.ts

```diff
@@ -561,6 +590,35 @@ export class Slider extends ControlElement {
     }
   }
 
+  /**
+   * Update the ARIA value text for a given thumb.
+   *
+   * @param thumbRef - The reference to the thumb element.
+   * @param markerValue - The value associated with the marker.
+   * @returns {void}
+   */
+  private updateAriaValueText(thumbRef: HTMLDivElement | undefined, markerValue: string): void {
+    if (!thumbRef) {
+      return;
+    }
+
+    const activeMarker = this.getActiveMarker(markerValue);
+
+    if (!activeMarker) {
+      thumbRef.removeAttribute('aria-valuetext');
+      return;
+    }
+
+    const markerLabel = activeMarker.textContent;
+    const ariaValueText = markerLabel ? `${markerLabel} ${markerValue}` : null;
```

Comment:
Let's simplify it with optional chaining.
```suggestion
    const markerLabel = activeMarker?.textContent;
    const ariaValueText = markerLabel ? `${markerLabel} ${markerValue}` : null;
```